hack/e2e/run.sh

```diff
@@ -50,14 +50,14 @@ IMAGE_TAG=${IMAGE_TAG:-${TEST_ID}}
 
 # kops: must include patch version (e.g. 1.19.1)
 # eksctl: mustn't include patch version (e.g. 1.19)
-K8S_VERSION=${K8S_VERSION:-1.20.8}
+K8S_VERSION=${K8S_VERSION:-1.22.3}
 
-KOPS_VERSION=${KOPS_VERSION:-1.21.0}
+KOPS_VERSION=${KOPS_VERSION:-1.23.3}
```

Comment:
Should this match the version of the cluster? I didn't before, so as long as the tests pass, maybe we're fine.

Reply:
yep good catch, this should match, revised

Reply:
Not sure this is the reason, looking at why efs-utils is not able to install

```
 #0 9.560 Error: Package: rpm-build-4.11.3-48.amzn2.0.2.x86_64 (amzn2-core)
#0 9.560            Requires: rpm = 4.11.3-48.amzn2.0.2
#0 9.560            Installed: rpm-4.11.3-48.amzn2.0.3.x86_64 (installed)
#0 9.560                rpm = 4.11.3-48.amzn2.0.3
#0 9.560            Available: rpm-4.11.3-25.amzn2.0.1.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-25.amzn2.0.1
#0 9.560            Available: rpm-4.11.3-25.amzn2.0.3.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-25.amzn2.0.3
#0 9.560            Available: rpm-4.11.3-40.amzn2.0.3.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-40.amzn2.0.3
#0 9.560            Available: rpm-4.11.3-40.amzn2.0.4.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-40.amzn2.0.4
#0 9.560            Available: rpm-4.11.3-40.amzn2.0.5.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-40.amzn2.0.5
#0 9.560            Available: rpm-4.11.3-40.amzn2.0.6.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-40.amzn2.0.6
#0 9.560            Available: rpm-4.11.3-48.amzn2.0.2.x86_64 (amzn2-core)
#0 9.560                rpm = 4.11.3-48.amzn2.0.2
#0 9.560  You could try using --skip-broken to work around the problem
#0 9.633 ** Found 2 pre-existing rpmdb problem(s), 'yum check' output follows:
#0 9.633 ca-certificates-2021.2.50-72.amzn2.0.5.noarch has missing requires of p11-kit >= ('0', '0.23.5', None)
#0 9.633 ca-certificates-2021.2.50-72.amzn2.0.5.noarch has missing requires of p11-kit-trust >= ('0', '0.23.5', None)
#13 ERROR: process "/bin/sh -c mkdir -p /tmp/rpms &&     if [ \"$EFSUTILSSOURCE\" = \"yum\" ];     then echo \"Installing efs-utils from Amazon Linux 2 yum repo\" &&          yum -y install --downloadonly --downloaddir=/tmp/rpms amazon-efs-utils-1.34.4-1.amzn2.noarch;     else echo \"Installing efs-utils from github using the latest git tag\" &&          yum -y install git rpm-build make &&          git clone https://github.com/aws/efs-utils &&          cd efs-utils &&          git checkout $(git describe --tags $(git rev-list --tags --max-count=1)) &&          make rpm && mv build/amazon-efs-utils*rpm /tmp/rpms &&          cd .. && rm -rf efs-utils &&          yum clean all;     fi" did not complete successfully: exit code: 1 
```
I dont see a straight correlation there